### ISSUE TITLE
[FR] Implement Controls for [Pi Camera Module 3]

### ISSUE THREAD

**Issue by andrewlockey** [The-EG/OctoPrint-CameraSettings#106]
**Camera manufacturer and model (ie. NexiGO N660)**: 
Pi Camera Module 3, 12 Megapixel Sony IMX708 image sensor, powered autofocus, improved low-light sensitivity and HDR 

**Unknown controls information (click 'Copy to Clipboard' on the warning and paste here)**:
{
  "hevc_sequence_parameter_set": {
    "type": "unknown"
  },
  "hevc_picture_parameter_set": {
    "type": "unknown"
  },
  "slice_param_array": {
    "type": "unknown"
  },
  "hevc_scaling_matrix": {
    "type": "unknown"
  },
  "hevc_decode_parameters": {
    "type": "unknown"
  }
}

**Comment by The-EG** [comment]
Hmm...These unknown controls should already be hidden. And I've definitely included the new PiCam3 in the latest RC.

What version of this plugin are you using?

**Comment by achampion** [comment]
Same for me: Camera Settings (0.4.1)

**Comment by The-EG** [comment]
> However, with both 0.4.1 and 0.4.2RC1 nothing is sticking, e.g. clicking off Vertical Flip (which defaulted to on), it just immediately jumps back on.

Can you please see if there is any errors in the browser console (press ctrl-shift-I) and also provide a debug log by adding a debug entry for this plugin in the 'Logging' section of OctoPrint settings, reproducing this behavior, and then uploading your octoprint.log here (also available on that logging settings page).

**Comment by achampion** [comment]
Yes, I am getting console areas for any interaction with the camera settings.

I get these 2 errors for any interaction with the camera settings, e.g. attempt to flip horizontal:
![image](https://user-images.githubusercontent.com/4431181/224548958-34599473-9376-4510-81c9-cee7936c404f.png)
![image](https://user-images.githubusercontent.com/4431181/224548906-ffe75a00-59c2-409a-ac71-1a36c0622ffb.png)

I also upgraded to 1.9.0RC2 and still no difference.

**Comment by MarvinLind** [comment]
This was resolved for me by 0.4.2RC1. I did however not have all of those "Unknown Controls" only "hevc_scaling_matrix" (I think).

**Comment by The-EG** [comment]
Thank you, finally someone confirmed! 😄
